saas/static/js/djaodjin-saas.js

```diff
@@ -385,7 +385,8 @@ CartItem.prototype = {
     };
 
     $.fn.invoice.defaults = {
-        currency_unit: "usd",
+        currency_unit: !globalSettings.default_unit ?
```

Comment:
It is OK to leave "usd" here and pass the default_unit when the jQuery component is initialized in the template.

        cardForm.find("#invoicables").invoice(
            {currency_unit: '{{lines_unit}}',
             saas_api_cart: "{{ urls.api_cart }}" });

lines_unit will always be initialized correctly by definition of how the template is constructed in the View.